client/src/ui/base/_themes.scss

```diff
@@ -328,15 +332,3 @@ body,
 .dark {
   @include dark-theme;
 }
-
-@media (prefers-color-scheme: light) {
```

Comment:
It looks quite strange, so I set the color scheme in the JS script directly.

Reply:
We prefer CSS over JS, if we can.

Apart from "looking strange", is there any disadvantage of doing this here?

Reply:
I'm confused by the current CSS approach in that CSS vars from `prefers-color-scheme` are targeted to the root element (`html`), then when dark mode is enabled, these CSS vars are overridden in `body` element. by adding `dark` class This is confusing and not at all flexible, given that `color-scheme` property only needs to be applied to the `html` element (it won't work for `body`), so I decided to use the JS approach.